subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -281,19 +632,49 @@ uint32_t ll_test_tx(uint8_t chan, uint8_t len, uint8_t type, uint8_t phy)
 	return 0;
 }
 
-uint32_t ll_test_rx(uint8_t chan, uint8_t phy, uint8_t mod_idx)
+uint32_t ll_test_rx(uint8_t chan, uint8_t phy, uint8_t mod_idx, uint8_t expected_cte_len,
+		    uint8_t expected_cte_type, uint8_t slot_duration, uint8_t switch_pattern_len,
+		    const uint8_t *ant_ids)
 {
 	uint32_t err;
+	bool cte_expected = (expected_cte_len > 0) ? 1 : 0;
 
 	if (!phy || (phy > 0x03)) {
```

Comment:
I was not part of the changes in the PR but please change magic number to a macro.

Reply:
Done